Improve silo example (#729)
lethe-cfd/lethe: doc/source/examples/dem/silo/silo.rst

Running the simulation
----------------------

> This simulation can be launched by (in parallel mode on 8 processes):

Reviewer: ```suggestion
This simulation can be launched in parallel (8 processes) running:
```

.. code-block:: text

  mpirun -np 8 dem_3d silo-Golshan.prm

.. warning::

> 	This example takes approximately 14 hours on 8 cores. This high computational cost is due to the long simulation time (30 s of real-time).

Reviewer: Maybe use "computational time" instead of "computational cost"?
